dafny-lang/dafny
# Fix migration of recently changed locations

### Changes
- Migration of recently changed locations wasn't working well. In particular `ChangedVerifiables` was not being migrated at all.
- This PR simplifies determination of verification order by removing `ChangedVerifiables` and only depending on `RecentChanges`, which was already being migrated.
- Fix a bug in the migration of `RecentChanges` that didn't account for Uris

### Testing
- Add a test for the migration of recently changed locations
- Change an existing test to expose the multiple file related bug that this PR fixes

<small>By submitting this pull request, I confirm that my contribution is made under the terms of the [MIT license](https://github.com/dafny-lang/dafny/blob/master/LICENSE.txt).</small>
